linkerd/system/Cargo.toml

```diff
@@ -14,4 +14,4 @@ tracing = "0.1"
 
 [target.'cfg(target_os = "linux")'.dependencies]
 libc = "0.2"
-procinfo = "0.4"
+procfs = "0.15.1"
```

Comment:
We're still not making use of non-default features. We're importing a bunch of additional dependencies based on default features (e.g. `chrono` and `gzip`) that we won't be using. Can we instead just use non-default here?